algorithms/a3c/main.py

```diff
@@ -40,80 +54,275 @@
 parser.add_argument('-sync', '--synchronous', dest='synchronous', action='store_true',
                     help='Useful for debugging purposes e.g. import pdb; pdb.set_trace(). '
                          'Overwrites args.num_processes as everything is in main thread. '
                          '1 train() function is run and no test()')
 parser.add_argument('-async', '--asynchronous', dest='synchronous', action='store_false')
 parser.set_defaults(synchronous=False)
 
+# experiment, environment and logging setting
+parser.add_argument('-eid', '--experiment-id', default=False,
+                    help='random or chosen guid for folder creation for plots and checkpointing.'
+                         ' If experiment folder and id taken, will resume training!')
+parser.add_argument('--seed', type=int, default=1,
+                    help='random seed (default: 1)')
+parser.add_argument('--test-sleep-time', type=int, default=200,
+                    help='number of seconds to wait before testing again (default: 200)')
+parser.add_argument('--checkpoint-freq', type=int, default=100000,
+                    help='number of episodes passed for resuming')
+parser.add_argument('--verbose-num-steps', default=False,
+                    help='print step number every args.num_steps')
+parser.add_argument('--num-steps', type=int, default=20,
+                    help='number of forward steps in A3C (default: 20)')
+
+# ai2thor settings
+parser.add_argument('--config-file-name', default='default_config.json',
+                    help='File must be in gym_ai2thor/config_files. Other good variants with '
+                         'natural language are: NL_pickup_5_objects_surround_fp_211_v0.1.json and'
+                         'NL_pickup_multiple_cups_only_fp404_v0.1.json')
+parser.add_argument('--resume-latest-config', default=1,
+                    help='Whether to resume latest_config.json found in experiment folder'
+                         'Default is set so user cannot override settings from text file')
+
+parser.add_argument('--max-episode-length', type=int, default=1000,
+                    help='maximum length of an episode (default: 1000000)')
+parser.add_argument('--num-random-actions-at-init', type=int, default=0,
+                    help='Number of random actions the agent does on initialisation')
+
 # Atari arguments. Good example of keeping code modular and allowing algorithms to run everywhere
 parser.add_argument('--atari', dest='atari', action='store_true',
                     help='Run atari env instead with name below instead of ai2thor')
 parser.add_argument('--atari-render', dest='atari_render', action='store_true',
                     help='Render atari')
 parser.add_argument('--atari-env-name', default='PongDeterministic-v4',
                     help='environment to train on (default: PongDeterministic-v4)')
-#
 parser.set_defaults(atari=False)
```

Comment:
Isn't the default already False? only if you `--atari` it will be true. Why doing it twice? same applied to atari_render